packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -394,14 +410,14 @@ const ruleset: any = {
     /// ARM RPC rules for Put patterns
     ///
 
-    // RPC Code: RPC-Put-V1-01
-    PathForPutOperation: {
-      description: "The path for 'put' operation must be under a subscription and resource group.",
+    // RPC Code: RPC-Put-V1-01, RPC-Get-V1-11
+    PathForResourceGroupOperation: {
+      description: "The path must be under a subscription and resource group.",
```

Comment:
>The path must be under a subscription and resource group [](http://example.com/codeflow?start=20&length=56)

The path must be under a subscription and resource group for tracked resource types.